src/tasks/models.py

```diff
@@ -11,7 +11,7 @@
 
 
 existing_tables = PostgresFetch(
-    name='query-existing-tables',
+    name="query-existing-tables",
```

Comment:
Is this a black formatting thing?  They require `"` instead of `'`?

Reply:
I think in prefect source code, “ is preferred for Postgres functions

Reply:
I think in prefect source code, “ is preferred for Postgres functions